### ISSUE TITLE
Drag a draggable div outside of a parent div

### ISSUE THREAD

**Issue by YanivRan** [react-grid-layout/react-draggable#154]
I have a scrollable pane (div), with many objects inside. I also have another div (a table) that I should be able to drag the objects from the left div to the table. Right now it seems that once a draggable is inside a parent div that confines it, and if it's a scrollable one then it extends the scroll to infinity without the ability to drag the element anywhere else in the screen... 
As I need to clone the elements instead of dragging the original, is there a way to clone and create a draggable from the clone on mouse click? I guess I should use React.cloneElement, but not sure exactly how...

**Comment by YanivRan** [comment]
Ok, 
So... there are several issues that I've encountered, and I'm not sure how to go about it. If I have a div that has limited height and is scrollable I encounter the following issues, some I can resolve, but some serious ones I can, and I tried draggableCore and still don't know how I can use that to solve the issue:
1. In order to "drag" the div outside the enclosing div, I had to change it's position to absolute. This works fine but you of course lose style of the div if it's based on the parent div. In order to solve this, I have a state that if the div is dragged it will set the "original" style back to the dragged div. 
2. Once I drop the div, and the position goes back to the "original" position (relative), then it "hided" somewhere in the parent div. I needed to reset the x and y states of the draggable component to put it back where it started.
3. the biggest issue, which I don't have a solution to yet, is when dragging, it doesn't really makes the right calculation of the location in this case. If I scroll down in the original div, the div selected will start somewhere down the screen and not where the mouse is. 

Any ideas how to deal with these issues the best, would be appreciated. I think it would be nice to have a ref to the enclosed draggableCore component, or at least being able to set the start x and y positions? I see there's a start prop, but not sure if I can use that? couldn't find much about that in the docs..
